changelog.d/20250311_061256_kurtmckee_resolve_deprecation_warning_issue_117.rst

```diff
@@ -0,0 +1,5 @@
+Fixed
+.....
+
+- Add a ``pyproject.toml`` file to resolve a ``DeprecationWarning``
+  when installing the project in editable mode.
```

Comment:
As with other PRs, I wouldn't include internal changes in the changelog.

Reply:
I can remove this. :+1: